docs/apache-hive-catalog.md

```diff
@@ -160,6 +152,21 @@ Hive automatically adds and manages some reserved properties. Users aren't allow
+| `location`          | The location for table storage, such as `/user/hive/warehouse/test_table`.                                                                 | HMS uses the database location as the parent directory by default.                                                                                  | No       | 0.2.0         |
+| `table-type`        | Type of the table. Valid values include `MANAGED_TABLE` and `EXTERNAL_TABLE`.                                                              | `MANAGED_TABLE`                                                                                                                                     | No       | 0.2.0         |
+| `format`            | The table file format. Valid values include `TEXTFILE`, `SEQUENCEFILE`, `RCFILE`, `ORC`, `PARQUET`, `AVRO`, `JSON`, `CSV`, and `REGEX`.    | `TEXTFILE`                                                                                                                                          | No       | 0.2.0         |
+| `input-format`      | The input format class for the table, such as `org.apache.hadoop.hive.ql.io.orc.OrcInputFormat`.                                           | The property `format` sets the default value `org.apache.hadoop.mapred.TextInputFormat` and can change it to a different default.                   | No       | 0.2.0         |
+| `output-format`     | The output format class for the table, such as `org.apache.hadoop.hive.ql.io.orc.OrcOutputFormat`.                                         | The property `format` sets the default value `org.apache.hadoop.hive.ql.io.HiveIgnoreKeyTextOutputFormat` and can change it to a different default. | No       | 0.2.0         |
+| `serde-lib`         | The serde library class for the table, such as `org.apache.hadoop.hive.ql.io.orc.OrcSerde`.                                                | The property `format` sets the default value `org.apache.hadoop.hive.serde2.lazy.LazySimpleSerDe` and can change it to a different default.         | No       | 0.2.0         |
+| `serde.parameter.`  | The prefix of the serde parameter, such as `"serde.parameter.orc.create.index" = "true"`, indicating `ORC` serde lib to create row indexes | (none)                                                                                                                                              | No       | 0.2.0         |
+| `serde-name`        | The name of the serde                                                                                                                      |                                                                                                                                                     | No       | 0.2.0         |
```

Comment:
Need some help, couldn't find any information about `serde-name`

Reply:
@mchades would you please help on this?

Reply:
FYI:
https://github.com/apache/gravitino/blob/7a42199a5fcaa02eaa8053cfec5b6f7b3dab3850/catalogs/catalog-hive/src/main/java/org/apache/gravitino/catalog/hive/HiveTablePropertiesMetadata.java#L96-L97

Reply:
Thanks